roles/core/precheck/defaults/main.yml

```diff
@@ -29,17 +29,17 @@ scale_reboot_automatic: false
 
 
 ## Whether or not to exchange SSH keys between nodes
-scale_prepare_exchange_keys: true
+scale_prepare_exchange_keys: false
 
 ## Path to public SSH key - will be generated (if it does not exist) and
 ## exchanged between nodes
 scale_prepare_pubkey_path: /root/.ssh/id_rsa.pub
 
 ## Whether or not enable SSH root login and pubkey authentication
-scale_prepare_enable_ssh_login: true
+scale_prepare_enable_ssh_login: false
 
 ## Whether or not to disable SSH hostkey checking
-scale_prepare_disable_ssh_hostkeycheck: true
+scale_prepare_disable_ssh_hostkeycheck: false
```

Comment:
Can you also set line 50 to false?  

This will address issue #11 and if we merge this PR, we can close #13

Reply:
done.